lib/util/page-number-functions.js

```diff
@@ -0,0 +1,71 @@
+const { removeLeadingWhitespaces, removeTrailingWhitespaces, isNumber } = require('./string-functions')
+
+const searchRange = (numerator, denominator, length) => {
```

Comment:
jsdoc would be helpful here

Reply:
Do you mean to comment on what the functions in page-number-functions do in jsdoc format?

Reply:
annotate the functions with jsdoc, as such:
```js
/**
 * Does something etc etc
 * @param {type} optional but would be useful
 * ...
 */
const searchRange = (...)